src/styles/index.scss

```diff
@@ -1,3 +1,114 @@
-body {
+$text-color: #616070;
+$title-color: #060b35;
+$color-button: #00acdc;
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+
+  font-family: 'Roboto', sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), you should avoid using universal selectors. It's better to use specific selectors to avoid unexpected behavior.